src/security/cert_generators/file/security_file_certgen.8.in

```diff
@@ -3,48 +3,49 @@
 .SH NAME
 security_file_certgen \- SSL certificate generator for Squid.
 .PP
-Version 1.0
+Version 1.1
 .
 .SH SYNOPSIS
 .if !'po4a'hide' .B security_file_certgen
-.if !'po4a'hide' .B [\-dhv]
-.br
-.if !'po4a'hide' .B security_file_certgen
-.if !'po4a'hide' .B "[\-d] \-s "
+.if !'po4a'hide' .B "[\-cdhv] [\-s "
 directory
-.if !'po4a'hide' .B "[\-M "
+.if !'po4a'hide' .B "\-M "
 size
 .if !'po4a'hide' .B "] [\-b "
 fs_block_size
 .if !'po4a'hide' .B ]
-.br
-.if !'po4a'hide' .B security_file_certgen
-.if !'po4a'hide' .B "[\-d] \-c \-s "
-directory
 .
 .SH DESCRIPTION
 .B security_file_certgen
 is an installed binary.
 .PP
 Because the generation and signing of SSL certificates takes time
-Squid must use external process to handle the work.
+Squid can use this helper as an external process to handle the work.
 .
-This process generates new SSL certificates and uses a disk cache of certificates
-to improve response times on repeated requests.
 Communication occurs via TCP sockets bound to the loopback interface.
 .
+This helper can use a disk cache of certificates to improve response
+times on repeated requests. It can also operate without disk storage
```

Comment:
Please consider this polishing (needs reformatting):

```
... It can also operate without a cache,
generating a new certificate on every request.
```

Reply:
done